activerecord/lib/active_record/destroy_association_async_job.rb

```diff
@@ -11,10 +11,14 @@ class DestroyAssociationAsyncJob < ActiveJob::Base
     discard_on ActiveJob::DeserializationError
 
     def perform(
-      owner_model_name: nil, owner_id: nil,
-      association_class: nil, association_ids: nil, association_primary_key_column: nil,
-      ensuring_owner_was_method: nil
-    )
+          owner_model_name: nil,
```

Comment:
The indentation is now wrong.

Reply:
Rubocop does not complain about it:

```
rails λ bundle exec rubocop -A
Inspecting 3003 files
...........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................

3003 files inspected, no offenses detected
```

What is the Rails' rule here?

Reply:
6 spaces, as before?